melatonin/internal/implementations.h

```diff
@@ -82,19 +82,19 @@ namespace melatonin::blur
 #elif defined(MELATONIN_BLUR_IPP)
         ippVectorSingleChannel (img, radius);
 #else
-        melatonin::blur::juceFloatVectorSingleChannel (img, radius);
+        melatonin::blur::juceFloatVectorSingleChannel (img, static_cast<int>(radius));
 #endif
     }
 
-    static void argb (juce::Image& srcImage, juce::Image& dstImage, size_t radius)
+    static void argb ([[maybe_unused]] juce::Image& srcImage, juce::Image& dstImage, size_t radius)
```

Comment:
This one is tripping me out, don't think I've ever had to annotate an argument with [[maybe_unused]]!